Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -551,7 +551,8 @@ private Tuple<StreamData, bool> OnSendToServer(StreamData data) {
                                 capabilities["workspace"]["configuration"] = true;
                                 
                                 var folders = GetFolders();
-                                Debug.Assert(folders.Any(), "no workspace or projects found");
+                                // Debug.Assert(folders.Any(), "no workspace or projects found");
+                                // This is allowed with say a loose file.
```

Comment:
i'll look into this bug

Reply:
That's the assert failure I've been seen, it repros even before opening any loose files. But for some reason I can't get it to repro anymore.